integration-cli/docker_cli_events_unix_test.go

```diff
@@ -63,43 +54,30 @@ func (s *DockerSuite) TestEventsOOMDisableFalse(c *check.C) {
 	go func() {
 		defer close(errChan)
 		out, exitCode, _ := dockerCmdWithError("run", "--name", "oomFalse", "-m", "10MB", "busybox", "sh", "-c", "x=a; while true; do x=$x$x$x$x; done")
-		if expected := 137; exitCode != expected {
-			errChan <- fmt.Errorf("wrong exit code for OOM container: expected %d, got %d (output: %q)", expected, exitCode, out)
-		}
+		expected := 137
```

Comment:
This one should not replace this one as it's used on next `select` with the chan `errchan`